content/docs/user-guide/dvc-files.md

```diff
@@ -143,9 +143,12 @@ stages:
     metrics:
       - performance.json
   training:
-    desc: Training stage description
-    cmd: python train.py
+    desc: Train model with Python
+    cmd:
+      - pip install -r requirements.txt
+      - python train.py --out ${model_file}
     deps:
+      - requirements.txt
       - train.py
```

Comment:
This example is not great...

Reply:
This was introduced in https://github.com/iterative/dvc.org/pull/1980/files#diff-101f4b5763f1f071bfa5c855cf2ed36fa5be72531acaa2c0beb2245da215c6d6 so I'm not sure why it's not already in master. Probably a bad merge somewhere...